error_test.go

```diff
@@ -45,10 +43,158 @@ func TestError(t *testing.T) {
 			if tc.err.Error() != tc.expected {
 				t.Errorf("expected err.Error() to be '%s' but was '%s'", tc.expected, tc.err.Error())
 			}
-			uuidErr := &ErrUUID{}
+			var uuidErr Error
 			if !errors.As(tc.err, &uuidErr) {
 				t.Error("expected errors.As() to work")
 			}
 		})
 	}
 }
+
+func TestAllErrorMessages(t *testing.T) {
```

Comment:
This test is testing several different operations (`Parse()`, `Scan()`, `UnmarshalBinary()`, `FromStringOrNil()`, and several `TimestampFromV*` variants).  A given test should be testing one thing so this should be broken apart IMO.

Reply:
It's just testing the error messages, not the operations/functions themselves. So I think this is better.